app-commons/src/main/scala/org/bitcoins/commons/jsonmodels/dlc/DLCSignatures.scala

```diff
@@ -1,36 +1,50 @@
 package org.bitcoins.commons.jsonmodels.dlc
 
+import org.bitcoins.core.protocol.script.P2WPKHWitnessV0
 import org.bitcoins.core.protocol.tlv.FundingSignaturesV0TLV
 import org.bitcoins.core.protocol.transaction.TransactionOutPoint
 import org.bitcoins.core.psbt.InputPSBTRecord.PartialSignature
-import org.bitcoins.core.util.MapWrapper
+import org.bitcoins.core.util.SeqWrapper
 import org.bitcoins.crypto.{ECAdaptorSignature, Sha256Digest}
 
 sealed trait DLCSignatures
 
 case class FundingSignatures(
-    sigs: Map[TransactionOutPoint, Vector[PartialSignature]])
-    extends MapWrapper[TransactionOutPoint, Vector[PartialSignature]]
+    sigs: Vector[(TransactionOutPoint, Vector[PartialSignature])])
+    extends SeqWrapper[(TransactionOutPoint, Vector[PartialSignature])]
```

Comment:
I feel like there should be a way to keep this as a `Map` so we don't lose out on performance

Reply:
There is a way (namely either doubling up and having both a Vector and a Map, or adding indices to the values in the map) but I don't think it is worth it for funding signatures (as opposed to CETSignatures) because there will never be a large number so the change in performance is negligible